iocBoot/iocBPMRFFEApp/save_restore.cmd

```diff
@@ -0,0 +1,27 @@
+# Save and restore setup
+
+save_restoreSet_status_prefix("${P}${R}")
+# Debug-output level
+save_restoreSet_Debug(0)
+
+# Is it okay to save/restore sets with missing values (no CA connection to PV)?
+save_restoreSet_IncompleteSetsOk(1)
+# Save dated backup files?
+save_restoreSet_DatedBackupFiles(1)
+
+# Number of sequenced backup files to write
+save_restoreSet_NumSeqFiles(3)
+# Time interval between sequenced backups
+save_restoreSet_SeqPeriodInSeconds(300)
+
+# Specify where save files should be
+set_savefile_path("/var/opt/rffe-epics-ioc", "autosave")
+
+# Specify what save files should be restored.  Note these files must be
+# in the directory specified in set_savefile_path(), or, if that function
+# has not been called, from the directory current when iocInit is invoked
+set_pass0_restoreFile("$(P)$(R).sav")
```

Comment:
Follow the autosave pattern and use `$(P)$(S)_settings.sav`

But you can also remove all comments from this file.